### ISSUE TITLE
[UX] Improve term visibility condition 

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3634]
The option for term id always throws me off, I never expect `Taxonomy term: TID`

I would prefer either `Taxonomy: Term ID` or  `Taxonomy Term: ID`

<img width="618" alt="Screen Shot 2019-03-29 at 9 57 36 AM" src="https://user-images.githubusercontent.com/397895/55249358-1ad71f80-5209-11e9-9ddc-ad33b18f653c.png">

PR: https://github.com/backdrop/backdrop/pull/2604

**Comment by laryn** [comment]
I think in Views it will show this way: "Taxonomy term: Term ID" which is a bit redundant, but better than TID.

**Comment by jenlampton** [comment]
That works too!  In views I think the labels are assembled with "entity type" then "property name" so maybe we should match that here.

**Comment by nicolascrossi** [comment]
I'm filing a pull request that changes "Taxonomy term TID is" to "Taxonomy term ID is". Changing the other values in the window is harder, since the value is built by concatenating various variables.

**Comment by quicksketch** [comment]
Thanks @nicolascrossi! That seems like an improvement as well. The original request here was to change the option in the "Visbility condition" select list, rather then the label of the text field. We can change both places though.

The option list is ultimately generated in `layout_layout_access_info()` in this line:
```
'title' => t('@entity: @entity_id_key', array('@entity' => $entity_info['label'], '@entity_id_key' => $key_label)),
```

So if we change "Taxonomy term: TID", it's also going to affect all other entity types, such as "Node: NID" will become "Node: ID" and "User account: UID" is going to become "User account: ID".

I'm not 100% certain this would be an improvement. Is specifying what kind of ID it is providing additional clarity?

**Comment by jenlampton** [comment]
> Is specifying what kind of ID it is providing additional clarity?

I don't think it is. I actually thought saying something other than simply `ID` might be adding confusion. TID is an abbreviation for `Term ID`, so saying `Term TID` is really like saying `Term Term ID`, which is weird. 

Additionally terms, for example, do not have an NID or a UID, so there's no need to specify which ID we are talking about. There is only one :)

People who are new to Drupal/Backdrop will not necessarily know that they need when they a TID vs an NID, but they'll know they want an ID of some kind.
